src/languageservice/parser/jsonParser07.ts

```diff
@@ -795,6 +796,8 @@ function validate(
           validate(mustMatchYamlProp, typeSchemaProp, subSchema, subValidationResult, subMatchingSchemas, options);
           if (
             !subValidationResult.hasProblems() ||
+            // allows some of the other errors like: patterns validations
+            subValidationResult.enumValueMatch ||
```

Comment:
take an alternative if the const or enum value match
 - ignore other errors in this situation